app/assets/stylesheets/components/_auction-show.scss

```diff
@@ -121,15 +120,6 @@
   margin-bottom: $base-padding-lite;
   margin-top: 1rem;
 
-  a {
-    text-decoration: underline;
-
-    &:hover,
-    &:focus {
-      text-decoration: none;
```

Comment:
yes! we are moving away from underlined links and toward links with no text decoration unless you hover (which is default for standards)